### PR TITLE
OCPBUGS-985: Revert "Test Baremetal deployment without a provisoning network"

### PR DESCRIPTION
**Pull request by elfosardo**
This reverts commit d4a2156569ad4c68487eef832373b4a4a6384c50.

Reverts #26676

### CODE REVIEW

**Comment by stbenjam** [comment]
You have the option to add the test to the Broken list here if you'd prefer: https://github.com/openshift/origin/blob/master/test/extended/util/annotate/rules.go#L31

When the test went through presubmits, wasn't it never run? Or did it break after? I can't tell but I don't see it in the e2e-metal-ipi-serial run on the original PR.

**Comment by elfosardo** [comment]
> You have the option to add the test to the Broken list here if you'd prefer: https://github.com/openshift/origin/blob/master/test/extended/util/annotate/rules.go#L31
> 
> When the test went through presubmits, wasn't it never run? Or did it break after? I can't tell but I don't see it in the e2e-metal-ipi-serial run on the original PR.

@stbenjam it's not in the original PR as it should run on virtualmedia only as the Provisioning Network should be Disabled, and it was not run there.....
locally it passed as far as I remember

I'll add them to broken then, thanks!

**Comment by elfosardo** [comment]
nvm, we'll remove them and redo them from scratch

File: test/extended/util/annotate/generated/zz_generated.annotations.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package generated
 
 import (
 	"fmt"
+
```
Comment (stbenjam) [inline test/extended/util/annotate/generated/zz_generated.annotations.go:5]: It looks like `make verify`  is unhappy about this

Comment (elfosardo) [reply test/extended/util/annotate/generated/zz_generated.annotations.go]: it was my editor taking not needed initiatives

**Pull request merged**